src/EPR.Calculator.API/Constants/ErrorMessages.cs

```diff
@@ -8,5 +8,7 @@ public static class ErrorMessages
         public static readonly string YearRequired = "Parameter Year is required";
         public static readonly string SchemeParameterTemplateValuesMissing =
             $"Uploaded Scheme parameter template should have a count of {DefaultParameterUniqueReferences.UniqueReferences.Length}";
+        public static readonly string LapcapDataTemplateValuesMissing =
+            $"Uploaded Scheme parameter template should have a count of {LapcapDataUniqueReferences.UniqueReferences.Length}";
```

Comment:
The text should refer to lapcap instead of scheme parameter.

Reply:
Done